import { LedgerCustom, LedgerError, LedgerErrorReason } from '../common';
/**
 * Error thrown when the intent received in entry can’t be recognized or correlated in the bridge.
 * It can mean that it’s not signed by expected key or the intent reference doesn’t exist in the bridge system.
 */
export declare class BridgeLedgerFailed extends LedgerError {
    /**
     *
     * @param detail error detail - message
     * @param custom custom data record to attach
     *               additional info about the error
     */
    constructor(detail: string, custom?: LedgerCustom);
    static REASON: LedgerErrorReason;
}
